client/components/search/index.jsx

```diff
@@ -38,7 +38,8 @@ module.exports = React.createClass( {
 		onKeyDown: React.PropTypes.func,
 		disableAutocorrect: React.PropTypes.bool,
 		onBlur: React.PropTypes.func,
-		searching: React.PropTypes.bool
+		searching: React.PropTypes.bool,
+		forceCloseButton: React.PropTypes.bool
```

Comment:
Uhm there might be some confusion here. Maybe we aren't talking about the same thing? 

Making a step back: if I look at [this design](https://cloud.githubusercontent.com/assets/191598/11665536/da9fdf3e-9db5-11e5-8798-c1f6f3301006.png) from above, there's no `×`, and I expect it to appear only if I type something. Which is the current behaviour, and doesn't need forcing it.

If that's correct... what's the need of a prop to force it visible? And once it's visible and on an empty content block, what would it do?